ui/src/components/Header/index.tsx

```diff
@@ -1,78 +1,81 @@
-  className?: string
+    className?: string
+    handleDrawerOpen: () => void,
 }
 
-const Header = ({ className }: Props) => {
-  const { accountList } = useAccounts()
+const Header = ({className, handleDrawerOpen}: Props) => {
+  const {accountList} = useAccounts()
 
   return (
-    <AppBar className={className}>
-      <Container maxWidth="xl">
-        <Toolbar disableGutters>
-          <Typography
-            variant="h6"
-            noWrap
-            component="div"
-            sx={{ flexGrow: 1 }}
-          >
+    <MuiAppBar position="fixed" className={className}>
+      <Toolbar>
+        <TypographyStyled
+          variant="h6"
+          noWrap
+        >
             Multix
-          </Typography>
-          {!!accountList?.length && (
-            <>
-              <Box className="buttonContainer">
-                <Button
-                  component={Link}
-                  to="/"
-                  className="buttonHeader"
-                >
-                  Home
-                </Button>
-                <Button
-                  component={Link}
-                  to="/create"
-                  className="buttonHeader"
-                >
-                  New Multisig
-                </Button>
-                <Button
-                  component={Link}
-                  to="/about"
-                  className="buttonHeader"
-                >
-                  About
-                </Button>
-                <Button
-                  component={Link}
-                  to="/help"
-                  className="buttonHeader"
-                >
-                  Help
-                </Button>
-              </Box>
-              <MultiProxySelection />
-            </>
-          )}
-          {/* <NetworkSelection /> */}
-        </Toolbar>
-      </Container>
-    </AppBar>
+        </TypographyStyled>
+        {!!accountList?.length && (
+          <BoxStyled>
+            {ROUTES.map(({url, name}) => (
+              <Button
+                component={Link}
+                to={url}
+                className="buttonHeader"
+              >
+                {name}
+              </Button>
+            ))}
+          </BoxStyled>
+        )}
+        <MultiProxySelection/>
+        {/*/!*<NetworkSelection />*!/*/}
+        <IconButtonStyled
+          color="inherit"
+          aria-label="open drawer"
+          edge="end"
+          onClick={handleDrawerOpen}
+        >
+          <MenuIcon/>
+        </IconButtonStyled>
+      </Toolbar>
+    </MuiAppBar>
   )
 }
 
-export default styled(Header)(({ theme }) => `
-  .buttonContainer {
-    flex-grow: 1;
-    display: flex;
-  }
+const BoxStyled = styled(Box)(({theme}) => `
+    display: none;
+    
+    @media (min-width: ${theme.breakpoints.values.sm}px) {
+        flex-grow: 1;
+        display: flex;
+    }
+`)
+
+
+const TypographyStyled = styled(Typography)`
+  flex-grow: 1;
+`
+
+const IconButtonStyled = styled(IconButton)(({theme}) => `
```

Comment:
Same comment as for the Typography component, I think we can go for all classes.
Also for this one, I noticed a glitch where for the first time you're visiting Multix, when you haven't connected your account yet, there's no menu available, so there should be no icon either. This button should be conditioned to `!!accountList?.length` and to avoid the repetition, we should go with
```js
const isShowMenu = useMemo(() => !!accountList?.length ,[accountList])
```

to avoid this
![image](https://user-images.githubusercontent.com/33178835/234407264-f4448e30-8368-4749-97f8-f92b9bd320db.png)